src/AppBundle/Service/ReportService.php

```diff
@@ -14,25 +15,48 @@
 use Doctrine\ORM\EntityManager;
 use Doctrine\ORM\EntityRepository;
 use Doctrine\ORM\Internal\Hydration\IterableResult;
+use Doctrine\ORM\QueryBuilder;
 
 class ReportService
 {
     /** @var EntityRepository */
     protected $reportRepository;
 
     /**
-     * @var CasRecRepository
+     * @var EntityManager
+     */
+    protected $_em;
```

Comment:
$em is already passed to the constructor

Reply:
and set as "_em " as a field property. I only added the missing declaration